helpers/sqs_helper.py

```diff
@@ -0,0 +1,108 @@
+"""
+Helper module for sqs messages
+"""
+import json
+import os
+import sys
+import time
+import boto3
+from botocore.exceptions import ClientError
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers.base_helper import BaseHelper
+from conf import sqs_conf
+
+class SqsHelper(BaseHelper):
+    """
+    SQS Helper object
+    """
+
+    def get_sqs_client(self):
+        """
+        Return sqs_client object
+        :param self:
+        :return sqs_client: SQS client object
+        """
+        try:
+            sqs_client = boto3.client('sqs', config=sqs_conf.config)
+            self.write(f'Created SQS client')
+        except ClientError as err:
+            self.write(f'Exception - {err}, Unable to create SQS client', level='error')
+        except Exception as err:
+            self.write(f'Unable to create SQS client, due to {err}', level='error') 
+
+        return sqs_client
+
+    def get_sqs_queue(self, queue_name):
```

Comment:
I feel like this is a redundant method. If we are getting the client, then we do not need this code at all.

Reply:
I meant `sqs_client` has its own `get_queue_url`: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.get_queue_url